packages/virtualized-lists/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@react-native/virtualized-lists",
+  "version": "1.0.0",
+  "description": "Virtualized lists for React Native.",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:facebook/react-native.git",
+    "directory": "packages/virtualized-lists"
+  },
+  "license": "MIT",
+  "peerDependencies": {
+    "react": "18.2.0",
+    "react-native": "*",
```

Comment:
This package is still tightly coupled to RN version, so we would need to do something more specific.

But we also have the issue where the main branch and other branches are unversioned. So whatever does the bumping really needs to be upping the peerDependencies of the monorepo packages to match the new RN version. @cortinico @cipolleschi will the current version bumping script bump monorepo packages peerDependencies on RN?

Reply:
I looked closer and I don't think the repo is set up to allow this yet. So feel free to disregard.

Reply:
> I looked closer and I don't think the repo is set up to allow this yet.

Correct. Currently the bumping is manual.
Also I would not set a peerDependency on `react` or `react-test-renderer` at all.

Reply:
@gabrieldonadel it looks like there is still a peerDependency on `react-test-renderer`. Could you change this to a devDependency?

Reply:
Sure @NickGerleman, I've just pushed a commit updating that